template_feed/Microsoft.DotNet.Common.ItemTemplates/content/WebConfig/web.config

```diff
@@ -4,13 +4,15 @@
   <!-- To customize the asp.net core module uncomment and edit the following section. 
   For more info see https://go.microsoft.com/fwlink/?linkid=838655 -->
   <!--
-  <system.webServer>
-    <handlers>
-      <remove name="aspNetCore"/>
-      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
-    </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
-  </system.webServer>
+  <location path="." inheritInChildApplications="false">
+    <system.webServer>
+      <handlers>
+        <remove name="aspNetCore"/>
+        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
+      </handlers>
+      <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
```

Comment:
@shirhatti @jkotalik The publish output has `processPath=dotnet` and `arguments=.\appname.dll`. Should we do something similar here? The `processPath` is easy, but the `appname` is harder since the item template doesn't really know what the name of the app is.

Reply:
Huh? Why would you want that. 

This item template in it's current incarnation works! The webSDK will substitute LAUNCHER_PATH and LAUNCHER_ARGS.

Reply:
Ok, that's what I wanted to clarify.